### DISCUSSION TITLE
Unable to import plugin netbox-topology_views

### DISCUSSION THREAD

**Discussion by Frank17489** [mattieserver/netbox-topology-views#518] [Q&A] [answered]
Hi,

I upgrade netbox to version 4.0.5 and it works
After that I try netbox_topology_views (4.0.0)

- source /opt/netbox/venv/bin/activate
- pip3 install netbox-topology_views
- insert "netbox-topology-views" in  /opt/netbox/local_requirements.txt
- change /opt/netbox/netbox/netbox/configuration.py                           PLUGINS = [ "netbox-topology_views" ]
- python3 manage.py migrate netbox_topology_views with

    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: Unable to import plugin netbox-topology_views: Module not found. Check that the plugin module has been installed within the correct Python environment.

any ideas?

thx Frank

**Comment by dreng** [comment]
"netbox-topology_views" doesn't look right. One underscore and one hyphen can't be correct. Typo?

**Comment by Frank17489** [comment, marked as answer]
thx, that was my fault. the package name has '-' in name , but the library has the underline.

;-(
